### PR TITLE
Move model type processing to Solver

### PR DESCRIPTION
**Pull request by l-bat**
### Changes

Mark nodes as ignored in Solver to reduce the log for the automatically generated ignored scope.

### Reason for changes

To improve user experience 

### Tests

conformance current log: 114
conformance with PR changes: 118

### CODE REVIEW

File: nncf/common/quantization/quantizer_propagation/solver.py [hunk 1]
```diff
@@ -333,6 +333,7 @@ def __init__(
         run_consistency_checks: bool = False,
         quantize_outputs: bool = False,
         post_processing_marker_metatypes: List[OperatorMetatype] = None,
+        model_type_marker_metatypes: List[OperatorMetatype] = None,
```
Comment (kshpv) [inline nncf/common/quantization/quantizer_propagation/solver.py:336]: Can we use more generic name that is not depending only on ModelType?
Maybe `metatypes_to_ignore`?

**Review by alexsu52 — APPROVED** [review]
LGTM

File: nncf/quantization/algorithms/min_max/backend.py [hunk 2]
```diff
@@ -209,13 +208,13 @@ def should_quantize_weight(weight_name: str, quantized_weight_names: Set[str]) -
 
     @staticmethod
     @abstractmethod
-    def get_ignored_scope(model_type: ModelType, device: TargetDevice) -> IgnoredScope:
+    def get_ignored_scope(model_type: ModelType, device: TargetDevice) -> List[OperatorMetatype]:
```
Comment (kshpv) [inline nncf/quantization/algorithms/min_max/backend.py:211]: I would update a name to something like `get_ignored_metatypes`

**Review by kshpv — CHANGES_REQUESTED** [review]

**Comment by l-bat** [comment]
run pytorch pre-commit tests

**Review by kshpv — APPROVED** [review]

**Review by vshampor — APPROVED** [review]

**Pull request merged**